.circleci/config.yml

```diff
@@ -710,6 +710,9 @@ commands:
           gcloud-service-key: <<parameters.gcloud-service-key>>
           google-compute-zone: <<parameters.google-compute-zone>>
           google-project-id: <<parameters.google-project-id>>
+      - run:
+          command: gcloud components install gke-gcloud-auth-plugin
```

Comment:
Kicked off CI in an effort to maybe save you time, now it says
```
ERROR: (gcloud.components.install) The following components are not available for your current SDK version [gke-gcloud-auth-plugin]. Please run `gcloud components update` to update your SDK.
```

Reply:
The Google Cloud SDK also needs to be upgraded. Though I tried that and then it still doesn't find the plugin for some reason.